### PR TITLE
Use EC2 instance ID for the initial super user password

### PR DESCRIPTION
**Pull request by mikeshultz**
n/t

### CODE REVIEW

File: backend/routes/auth.js [hunk 1]
```diff
@@ -45,6 +45,20 @@ async function getGcpSuperAdminPassword() {
   return password
 }
 
+/**
+ * Gets the EC2 instance ID
+ */
+async function getEC2InstanceID() {
+  const url = 'http://169.254.169.254/latest/meta-data/instance-id'
```
Comment (franckc) [inline backend/routes/auth.js:52]: is that IP guaranteed to never change?

File: backend/routes/auth.js [hunk 2]
```diff
@@ -262,6 +277,30 @@ module.exports = function (router) {
       }
     }
 
+    if (process.env.AWS_MARKETPLACE_DEPLOYMENT) {
+      /**
+       * In the case of a AWS marketplace deployment, the super-admin password
+       * is the EC2 instance ID.
```
Comment (franckc) [inline backend/routes/auth.js:283]: I'm not familiar with EC2. I assume there is no possibility an attacker could guess the instance ID?

File: backend/routes/auth.js (hunk 1 above)
Comment (mikeshultz) [reply backend/routes/auth.js:52]: From what I can tell, yes

File: backend/routes/auth.js (hunk 2 above)
Comment (mikeshultz) [reply backend/routes/auth.js:283]: Not that I'm aware of.  Though I do think these should only be temporary passwords(GCP included).  Perhaps that's something we can fix forward.

Comment (franckc) [reply backend/routes/auth.js:283]: Yep agreed as a separate feature we should encourage (or maybe force) the admin to provide a new password the first time they login.

Comment (mikeshultz) [reply backend/routes/auth.js:283]: See #473

**Pull request merged**